lib/codecs/CryptosuiteDecoder.js

```diff
@@ -18,7 +18,7 @@ export class CryptosuiteDecoder extends CborldDecoder {
     }
 
     // handle compressed cryptosuite string
-    const str = this.reverseAppContextMap.get(value) ??
+    const str = this.reverseCompressionMap.get(value) ??
       CRYPTOSUITE_ID_TO_STRING.get(value);
```

Comment:
I had to pause here and wonder why we're still using `CRYPTOSUITE_ID_TO_STRING` here at all -- my mental model would be that in legacy mode we would have put this into the type-specific decompression table and so it wouldn't be used here at all. Maybe this falls out from allowing a different type-specific decompression table when in legacy mode -- when we shouldn't allow that? This might be related to my comment earlier about that on the CHANGELOG/general API design.

Reply:
This is not needed here - removed.